yaml/boundaries.yaml

```diff
@@ -1,75 +1,79 @@
 filters:
+
+  # osm
   - filter:
       admin_level: ['2', '4']
       boundary: administrative
       'tags->boundary:type': aboriginal_lands
     min_zoom: 8
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: {col: admin_level}}
```

Comment:
We used to have a transform in python, but that got removed when we moved to the yaml syntax. I updated this to set the value explicitly to 3 in 4958518.